next/src/components/Event.tsx

```diff
@@ -4,29 +4,57 @@ import Card from '@mui/material/Card';
 import CardActionArea from '@mui/material/CardActionArea';
 import CardContent from '@mui/material/CardContent';
 import Box from '@mui/material/Box';
-import TranscribeButton from 'components/TranscribeButton';
 import { EventProps } from 'types/EventProps';
+import { useState } from 'react';
+import Transcript from 'components/Transcript'
+import TranscribeButton from 'components/TranscribeButton';
+import ErrorMessage from './ErrorMessage';
 
 type Props = { event: EventProps }
 
 export default function Event(props: Props) {
+  const [transcript, setTranscript] = useState(props.event.transcript)
+  const [buttonDisplay, setButtonDisplay] = useState(transcript === "")
```

Comment:
```suggestion
  const isTranscript = transcript === ""
```
useState で管理しなくても、既存の State から状態を参照するようにしてあげると良さそうかもですかね🙌

Reply:
たしかに、わざわざ useState 使わなくて良いですね。useState 増えると把握しづらいですし。
変更してみます〜。